test/errors/tupleExpandNonTuple-detailed.good

```diff
@@ -0,0 +1,9 @@
+─── error in tupleExpandNonTuple.chpl:3 [TupleExpansionNonTuple] ───
+  Cannot apply tuple expansion to non-tuple argument.
```

Comment:
`x` isn't really an argument here, it's a variable/an expression.  "Cannot apply tuple expansion to a non-tuple expression" (or "to an expression of non-tuple type"?) seems more accurate.